### ISSUE TITLE
Volla 22, VOS 14 - Battery loses capacity even when not in use

### ISSUE THREAD

**Issue by Kati1005** [HelloVolla/volla-os-beta-test#204]
Volla Phone 22
Version 14-20241124-STABLE-mimameid
Launcher Trebuchet
No security mode, no VPN
No running background processes
Use two sim cards + memory card

I have already asked the Volla user community via Telegram, who advised me to make a request here after unfortunately unsuccessful help

I bought the Volla 22 new in September 2024. Since November, the battery has been losing capacity even without use. This already started under VOS 13.
I bought a new battery to rule out that this was the cause, but without any change. 
An example: I charged it to 100 % in the evening and in the morning the battery only had 64 % without use ( about 9 pm to 6:00 am).
In the battery consumption view, it is always the system app that consumes 99 %.

A test with energy and data saving mode switched on and mobile data and Wi-Fi switched off saved 10 %. However, 25% of the battery was still consumed (Bluetooth is generally switched off).

The battery manager shows nothing.

I have reset the phone to factory settings in the meantime, with no result. 

I can also send an error report on request.

I have one more addition: I have noticed that the CPU utilization is very high. Attached a picture.

![Screenshot_20241215-065524_Trebuchet](https://github.com/user-attachments/assets/06c8a0cb-6484-4fd3-af97-dc6f2ff35814)
![Screenshot_20241215-065543_Trebuchet](https://github.com/user-attachments/assets/ee969fda-e987-42d8-a4a1-497100ecd88c)
![Screenshot_20241219-084421_Trebuchet](https://github.com/user-attachments/assets/478bdf18-4e2c-44d3-a25b-e3e7c2e1f5d2)

![Screenshot_20241226-091704_Trebuchet.png](https://github.com/user-attachments/assets/e45eb4fe-16fe-4544-8b25-6b0820576c47)

**Comment by wurzer** [comment]
After two full days in standby the battery drain is about 3%. I can't reproduce this issue. 

Your can block one app after another in the securoty mode to find the app or background process, that consumes the enegery. It's surprising, that two launchers are running. If you use Trebushet , you can block the Volla App or vice versa. 

Please check the battery consumption after updating to the Decemmber update an reopen the ticket, if a significant battery drain is still reproducable.

**Comment by Kati1005** [comment]
Thank you for your reply. How can I block the Volla Launcher in the security mode?

**Comment by Kati1005** [comment]
Information about my battery problem, which might help one or the other.


I had read in an Android forum, so not specifically Volla, that users also had such an enormous battery consumption and it was normal again by switching off the 4G calls (not to be confused with the 4G Internet. This is still active).


I tested that. So 4G calls off. The battery now lasted 70 hours with normal use from 100% to 10%. So almost three days. Then I charged the battery again and it lasted another three days. Then I fully charged it again and reactivated the 4G calls. It then went from 100% to 1% within 15 hours overnight and without any usage worth mentioning.


Then I deactivated the 4G calls again and fully charged it again. The battery lasted around three days again.
